### DISCUSSION TITLE
JustPlayer with Kodi

### DISCUSSION THREAD

**Discussion by billsmith564** [moneytoo/Player#97] [General]
Hi,

I'm currently using ExoPlayer (demo) to play Dolby Vision files on my Firestick 4k via Kodi (modified playercorefactory.xml)
ExoPlayer won't play files off my NAS which is a bit of an issue.

Now JustPlayer will play Dolby Vision files from my NAS via SolidFileExplorer but I can't get it to play with Kodi.

I've modified the playercorefactory.xml using the package name com.brouken.player but no dice.
i.e. <filename>com.brouken.player</filename>

while I can see ExoPlayer in the Android Data directory 'com.google.android.exoplayer2.demo' I can't see anything for JustPlayer.

Am I missing something here?

Mark.

**Comment by moneytoo** [comment]
Hi. See this: https://github.com/moneytoo/Player/issues/74#issuecomment-799657514
